.github/workflows/gemini-pr-review.yml

```
@@ -1,456 +1,137 @@
+            
+      - name: Check for required secrets
+        id: check_secrets
+        run: |
+          if [ -z "${{ secrets.GEMINI_API_KEY }}" ]; then
+            echo "::notice::GEMINI_API_KEY secret is not configured. Skipping AI review."
+            echo "skip=true" >> $GITHUB_OUTPUT
+            exit 0
+          fi
+          echo "skip=false" >> $GITHUB_OUTPUT
+
+      - name: Run Gemini review
```

actionlint (1.7.7)

step must run script with "run" section or run action with "uses" section

(syntax-check)